docs/modules/configuring/ref-configuring-proxy-example.adoc

```diff
@@ -0,0 +1,79 @@
+[id='ref-configuring-proxy-example-{context}']
+= Example Kroxylicious configuration
+
+[role="_abstract"]
+Kroxylicious configuration is defined in a `ConfigMap` resource.
```

Comment:
The (upstream) documentation can't assume Kubernetes. Kroxylicous can be run on bare metal or within Kubernetes.  On Kubernetes, using a ConfigMap is one way to the config it,

Reply:
Right. I think we can be upfront about this example showing configmap for deployment on kubernetes (which is script-replaced by `OpenShift` downstream). If we want more upstream examples, we can add them. 

So: "The following example shows configuration defined in a `ConfigMap` resource for Kroxylicious running on Kubernetes.
The `data` properties of the `ConfigMap` resource provide the following configuration:"

Reply:
I thought a bit more since my first comment.

If we starting showing a ConfigMap with Kroxylicious configuration then we'd need to show a Deployment that references the same configmap (and a Service...) for the whole thing to make sense.   I don't think the documentation is really ready for Kubernetization.  I think the documentation can only assume that that it is being on bare metal.  If the user wants to deploy to Kubernetes, then they are free to do that.

Obviously we can structure the documentation in such a way that derived works, which are platform opinionated, have an easy time repurposing the docs.

Reply:
Thanks @k-wall - how should we structure it to work upstream and downstream?